api/src/main/java/org/openmrs/api/db/hibernate/HibernateConceptDAO.java

```diff
@@ -1584,7 +1582,7 @@ public ConceptReferenceTerm getConceptReferenceTermByUuid(String uuid) throws DA
 	public List<ConceptReferenceTerm> getConceptReferenceTermsBySource(ConceptSource conceptSource) throws DAOException {
 		Criteria criteria = sessionFactory.getCurrentSession().createCriteria(ConceptReferenceTerm.class);
 		criteria.add(Restrictions.eq("conceptSource", conceptSource));
-		return (List<ConceptReferenceTerm>) criteria.list();
+		return criteria.list();
```

Comment:
i see lots of changes not about dropping the conditions on logging ,was this pr not about replacing conditional logging with slf4j parameters?